modules/apple_support/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "homepage": "https://github.com/bazelbuild/apple_support",
+    "maintainers": [
+        {
+            "email": "me@patrickbalestra.com",
```

Comment:
You can add me too if you'd like, I've been hoping to get to this but glad to see you did!

Reply:
Added you!